### PR TITLE
Implement Eldritch Invocation: Lifedrinker

### PR DESCRIPTION
**Pull request by Aeristoka**
As requested on Discord

### CODE REVIEW

File: src/common/settings.js [hunk 1]
```diff
@@ -555,6 +555,12 @@ const character_settings = {
         "description": "Your charisma and deity guide your attacks",
         "type": "bool",
         "default": false
+    },
+    "eldritch-invocation-lifedinker": {
+        "title": "Eldritch Invocation: Lifedrinker",
+        "description": "Your pact weapon drips with necrotic energry, lending extra damage to your strikes",
```
Comment (kakaroto) [inline src/common/settings.js:561]: typo `energry` => `energy`

Comment (Aeristoka) [reply src/common/settings.js]: fixed

**Comment by kakaroto** [comment]
yeay for conflicts :(

**Comment by Aeristoka** [comment]
> yeay for conflicts :(

So begins, once again, our silly dance of you merging my crap one at a time, and then me rebasing on my previously merged code, only to repeat it :D

**Comment by kakaroto** [comment]
What you mean silly dance? Is noble dance of ancestors!

**Pull request merged**